app/decorators/service_provider_session_decorator.rb

```diff
@@ -118,7 +118,8 @@ def url_options
   end
 
   def irs_attempts_api_session_id
-    @irs_attempts_api_session_id ||= request_url_params['irs_attempts_api_session_id']
+    @irs_attempts_api_session_id ||=
+      request_url_params['irs_attempts_api_session_id'] || request_url_params['tid']
```

Comment:
If we do end up needing this, we should confirm with the IRS that `tid` is what we think it is.